pom.xml

```diff
@@ -1,341 +1,398 @@
+				<executions>
+					<execution>
+						<id>copy-resources</id>
+						<!-- here the phase you need -->
+						<phase>site</phase>
+						<goals>
+							<goal>copy-resources</goal>
+						</goals>
+						<configuration>
+							<!-- plugin requires all resources to exist within resources/ directory -->
+							<!-- but we don't want to break links in documentation -->
+							<outputDirectory>${basedir}/wiki/resources/images</outputDirectory>
+							<resources>
+								<resource>
+									<directory>${basedir}/wiki/images</directory>
+								</resource>
+							</resources>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
 
-        </plugins>
-    </build>
+			<plugin>
+				<artifactId>maven-clean-plugin</artifactId>
+				<version>3.1.0</version>
+				<!-- We want to remove duplicate/copied resources from site generation -->
+				<configuration>
+					<filesets>
+						<fileset>
+							<directory>${basedir}/wiki/resources</directory>
+						</fileset>
+					</filesets>
+				</configuration>
+			</plugin>
+
+		</plugins>
+	</build>
+	<profiles>
+		<!-- The Configuration of the integration-test profile -->
+		<profile>
+			<id>integrationTest</id>
+			<properties>
+				<skipUTs>true</skipUTs>
+			</properties>
+			<activation>
+				<activeByDefault>false</activeByDefault>
+			</activation>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-failsafe-plugin</artifactId>
+						<version>2.22.0</version>
+						<configuration>
+							<skipTests>${skipTests}</skipTests>
+							<skipITs>${skipITs}</skipITs>
+							<systemPropertyVariables>
+								<rabbit.host>localhost</rabbit.host>
+								<rabbit.port>5672</rabbit.port>
+								<rabbit.username>myuser</rabbit.username>
+								<rabbit.password>myuser</rabbit.password>
+								<rabbit.exchange>ei-exchange</rabbit.exchange>
+								<rabbit.key>#</rabbit.key>
+							</systemPropertyVariables>
+							<excludes>
+								<exclude>**/systemtest/*</exclude>
+							</excludes>
+						</configuration>
+						<executions>
+							<execution>
+								<id>integration-test</id>
+								<goals>
+									<goal>integration-test</goal>
+									<goal>verify</goal>
+								</goals>
+								<phase>integration-test</phase>
+							</execution>
+						</executions>
+					</plugin>
+				</plugins>
+			</build>
+		</profile>
+
+		<!-- The Configuration of the system-test profile -->
+		<profile>
+			<id>systemTest</id>
+			<properties>
+				<skipUTs>true</skipUTs>
+			</properties>
+			<activation>
+				<activeByDefault>false</activeByDefault>
+			</activation>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-failsafe-plugin</artifactId>
+						<version>2.22.0</version>
+						<configuration>
+						<systemPropertyVariables>
+                            <propertiesFile>${systemtestsProperties}</propertiesFile>
```

Comment:
Where is this variable declared?

Reply:
I have created one.... see Tobias comments